doc/source/release-notes/Percona-Server-8.0.15-6.rst

```diff
@@ -43,12 +43,111 @@ New Features
 
 - RocksDB WAL file information can now be seen in the
   :table:`performance_schema.log_status` :ref:`table <log_status>`.
+
+- New :variable:`Audit_log_buffer_size_overflow` status variable has been
+  implemented to track when an :ref:`audit_log_plugin` entry was either
+  dropped or written directly to the file due to its size being bigger
+  than :variable:`audit_log_buffer_size` variable.
   
 
 Bugs Fixed
 ================================================================================
 
+- TokuDB and MyRocks native partitioning would never release memory which could
+  lead to a server crash. Bug fixed :psbug:`5508`.
+
+- using TokuDB or MyRocks native partitioning and ``index_merge`` could lead to
+  a server crash. Bugs fixed :psbug:`5206`, :psbug:`5562`.
+
+- upgrade from |Percona Server| 5.7.24 to :rn:`8.0.13-3` wasn't working with
```

Comment:
https://jira.percona.com/browse/PS-5223 also discusses a workaround in the upgrade guide which should be removed with this fix, please check